src/main/java/com/github/rodionovsasha/commentservice/AppConfig.java

```diff
@@ -0,0 +1,44 @@
+package com.github.rodionovsasha.commentservice;
+
+import com.github.rodionovsasha.commentservice.entities.Comment;
+import com.github.rodionovsasha.commentservice.entities.Topic;
+import com.github.rodionovsasha.commentservice.entities.User;
+import com.github.rodionovsasha.commentservice.repositories.CommentRepository;
+import com.github.rodionovsasha.commentservice.repositories.TopicRepository;
+import com.github.rodionovsasha.commentservice.repositories.UserRepository;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import static java.util.Arrays.asList;
+
+@Configuration
+@EnableSwagger2
+public class AppConfig {
+    public static final String API_BASE_URL = "/v1/api";
+
+    @Bean
+    CommandLineRunner runner(UserRepository userRepository, TopicRepository topicRepository, CommentRepository commentRepository) {
```

Comment:
What's this method for?

Reply:
load demo data before running

Reply:
let's do it within JFixtures?

Reply:
sure, it is just tmp data to speed up development